bodhi/tests/client/test___init__.py

```diff
@@ -392,6 +392,6 @@ def test_url_flag(self, send_request, query):
                 'close_bugs': True, 'stable_karma': None, 'csrf_token': 'a_csrf_token',
                 'staging': False, 'builds': u'nodejs-grunt-wrap-0.3.0-2.fc25', 'autokarma': True,
                 'edited': u'nodejs-grunt-wrap-0.3.0-2.fc25', 'suggest': None,
-                'notes': u'this is an edited note', 'request': None, 'bugs': u'',
-                'unstable_karma': None, 'type': 'bugfix'})
+                'notes': u'this is an edited note', 'notes_file': None,
+                'request': None, 'bugs': u'', 'unstable_karma': None, 'type': 'bugfix'})
         self.assertEqual(bindings_client.base_url, 'http://localhost:6543/')
```

Comment:
Hello @crungehottman!

No problem at all, I'm happy to help. To answer your first question, yes that is the correct file and class to put the test for the edit command in. Not all of bodhi's test follow this convention, but I try to name the test files after the file they are testing (i.e., ```bodhi/foo/bar.py```'s tests would be in a file called ```bodhi/tests/foo/test_bar.py```) and I like to have a class per "unit" I'm testing, where the ```edit()``` function would be a unit here.